static/src/js/containers/TimelineContainer/__tests__/TimelineContainer.test.tsx

```diff
@@ -144,7 +144,7 @@ describe('TimelineContainer component', () => {
 
   test('passes its props and renders a single Timeline component on the project page', () => {
     (useLocation as jest.Mock).mockReturnValue({
-      pathname: '/projects',
+      pathname: '/project',
```

Comment:
```suggestion
      pathname: 'routes.PROJECT',
```

Reply:
I'll make the update, but I didn't go through all the test files and make the routes changes, it would have probably been another 50 files changed

Reply:
Yea we can just do that over time when its noticed